### ISSUE TITLE
Show QR code when pressing `u` in console

### ISSUE THREAD

**Issue by amr3k** [svitejs/vite-plugin-qrcode#135]
### Describe the problem

Vite supports the `u` shortcut to show the URL.
It would be great if this plugins renders the QR code when vite displays URL.

### Describe the proposed solution

Show QR code whenever vite shows URL

### Alternatives considered

None

### Importance

nice to have

**Comment by bluwy** [comment]
I don't think we can support this without Vite exposing some APIs so we can add and integrate new shortcuts. Even with Vite 5 which exposes `server.bindCLIShortcuts`, it only works on server initialization so we can't add more to it. So I don't think this is something we can implement at the moment.
